### ISSUE TITLE
Rockbox PDA message seems wrong

### ISSUE THREAD

**Issue by goonstation-issuebot** [goonstation/goonstation#20083]
### Labels

[BUG][MINOR]

### Description

As CE, the message seems to not be formatted right

### Steps to reproduce

ROCKBOX&trade;-MAILBOT to mining, Sales Alert: Notification: 560 credits earned from Rockbox&trade; sale, deposited to your account.

### Expected Behavior

not being "ROCKBOX&trade;"

### Additional Information & Screenshots

Reported by: chaotictyrant
Client version: 515.1641
On server: Goonstation 3 Roleplay: Morty
Active test merges: #19063
Round log date: 2024-07-25-12-36
Reported on: 2024-07-25 13:34:56
Round ID: 184198
Map: NADIR

**Comment by TheTyrant25** [comment]
Specifically the error is the trademark appearing as
```
ROCKBOX&trade;-MAILBOT to mining, Sales Alert: Notification: 560 credits earned from Rockbox&trade; sale, deposited to your account.
```
and not correctly. This bug report's formatting seems to have undone my point

**Comment by JORJ949** [comment]
Fixed in faea489
